docs/src/contributing/overview.md

```diff
@@ -1,23 +1,22 @@
 # Contributor Overview
 
-Interested in contributing to the Deis project?  There are lots of ways to help.
+Interested in contributing to a Deis project?  There are lots of ways to help.
 
 ## File Bugs & Enhancements
 
-Find a bug? Want to see a new feature? Have a request for the maintainers? Open a [GitHub issue][issue] and we’ll get the conversation started.
+Find a bug? Want to see a new feature? Have a request for the maintainers? Open a Github issue in the applicable repository and we’ll get the conversation started.
 
 ## Write Documentation
 
-We are always looking to improve and expand our documentation. Most docs reside in the main repository under the [docs/][docs] directory. Simply fork the project, update docs and send us a pull request.
+We are always looking to improve and expand our documentation. Most docs reside in the deis/workflow repository under the [docs/][docs] directory. Simply fork the project, update docs and send us a pull request.
```

Comment:
They're in it... but I suppose that might not be the case since we'll publish these docs off to elsewhere.  I'll fix that.